src/modules/esl-traversing-query/core/esl-traversing-query.ts

```diff
@@ -96,7 +96,34 @@ export class TraversingQuery {
     return uniq(result);
   }
 
-  static traverse(query: string, findFirst: boolean, base?: Element | null, scope: Element | Document = document): Element[] {
+  /** Split multiple queries separated by comma (respects query brackets) */
+  public static splitQueries(str: string): string[] {
+    let last = 0;
+    let stack = 0;
+    const result = [];
+    for (let i = 0; i < str.length; i++) {
+      if (str[i] === '(') stack++;
+      if (str[i] === ')') stack = Math.max(0, stack - 1);
+      if (str[i] === ',' && !stack) {
+        result.push(str.substring(last, i).trim());
+        last = i + 1;
+      }
+    }
+    result.push(str.substring(last).trim());
+    return result;
```

Comment:
```suggestion
    const matches = str.split(/(?<!\([^\)]*),(?![^\(]*\))/g) || [];
    return matches.map((match) => match.trim());
```

Reply:
It's possible to solve the problem with regex

Reply:
This type of `RegExp` does not support by the Safari browser, so, unfortunately, we can not solve the nested term problems with RegExp cross-browser right now. The only thing that I recommend is to save your solution in the comment till the WebKit browser implements this feature
CC: @Natalie-Smirnova